guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,84 @@
+
+</div>
+
+{% include /tip/open.html label="Don’t: Hide the implications of a non-custodial wallet" icon="forbid" color="red" %}
+
+Within your onboarding process, be clear about what it means to have a non-custodial wallet. (and the responsibility that comes along with it). Within your onboarding process, be clear that “account” information cannot be recovered by your respective team.
+
+{% include /tip/close.html %}
+
+While this may sound daunting to first-time users, it’s crucial that these caveats are not hidden. Doing so may severely compromise the safety of their funds, as their default assumptions and behaviors could potentially mimic how they typically use centralized applications (for example, feeling like it is OK if they forget their account information because they can contact customer support to help them recover it). According to a study conducted by Ekandari et al about the usability of cryptocurrency wallets, it became clear that users struggled greatly with technical terminology and feeling as though they had a lack of guidance during wallet setup. By walking users through a short set of carousels that clearly frame these Bitcoin-specific features, users will feel well-equipped to confidently navigate and use your wallet.
+
+> Users reported that they often felt like they lacked guidance and understanding during wallet setup
+>
+> <cite>As researched by <a href="https://arxiv.org/pdf/1802.04351.pdf">Eskandari et al</a></cite>
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/first-onboarding-screen.png"
+   retina = "/assets/images/guide/onboarding/first-onboarding-screen@2x.png"
+   alt-text = "First onboarding screen"
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/second-onboarding-screen.png"
+   retina = "/assets/images/guide/onboarding/second-onboarding-screen@2x.png"
+   alt-text = "Second onboarding screen"
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/third-onboarding-screen.png"
+   retina = "/assets/images/guide/onboarding/third-onboarding-screen@2x.png"
+   alt-text = "Third onboarding screen"
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/fourth-onboarding-screen.png"
+   retina = "/assets/images/guide/onboarding/fourth-onboarding-screen@2x.png"
+   alt-text = "Fourth onboarding screen"
+%}
+
+</div>
+
+*This prototype exhibits the typical first screen that users are shown after downloading a Bitcoin wallet, followed by an outline of suggested informational carousels that explicitly lay out/guide users on the unique features of non-custodial wallets to make sure they understand these caveats. These screens suggest talking about ownership of their “account”, explicitly stating the recovery mechanism that the respective wallet uses, and security measures that they should take to protect their wallet.*
```

Comment:
Why is this text in italic at the bottom? Overall, the prototype doesn't seem well integrated into the page. The actual Figma prototype link appears in an image caption before a prototype is mentioned in the page content. I also didn't instantly connect the word prototype in this paragraph with the image sequence above (as that's just an image sequence and not a prototype). Would be great to clarify how this all ties together. If you want to highlight the prototype more, you could take the approach Daniel used in the case studies pages. There's a `prototype` include for this purpose, but you'd need to create that big preview image. Up to you.